manifests/user.pp

```diff
@@ -16,8 +16,9 @@
   $wrapper      = '/usr/bin/zpr_wrapper.py',
 ) inherits zpr::params {
 
-  # For placement of keys manually
+  include zpr::rsync_cmd
 
+  $permitted_commands = "${home}/.ssh/permitted_commands"
```

Comment:
It comes from manifests/rsync_cmd.pp. The reference here appears to be one that is no longer needed in this file and that I failed to remove.